multibody/kinematic/kinematic_evaluator_set.cc

```diff
@@ -271,6 +305,21 @@ int KinematicEvaluatorSet<T>::count_full() const {
   return count;
 }
 
+template <typename T>
+bool KinematicEvaluatorSet<T>::is_active(int index) const {
+  if (index < 0 || index >= count_full()) {
+    return false;
+  }
+  int count = 0;
+  for (const auto& e : evaluators_) {
+    if (index - count >= 0 && index - count < e->num_full()) {
```

Comment:
This block took a minute to understand - is it worth creating a separate vector that stores whether each line of all the evaluators are active? The only use case I can think of for creating such a vector is to be able to print out which constraints are active, thoughts?

Reply:
Like you said, I think that's the only real use case. I'd worry a little about ensuring that the vector stays synchronized. For instance, it could not be a property of the `Set` since the individual `Evaluators` can be updated without the `Set` knowing. I added a comment to explain what the code is doing. Let me know if that helps.